test/unit/org/apache/cassandra/config/ParseAndConvertUnitsTest.java

```diff
@@ -104,7 +104,8 @@ public void testConfigurationLoaderParser()
         assertNull(config.file_cache_size);
         assertNull(config.index_summary_capacity);
         assertEquals(new DataStorageSpec.LongMebibytesBound(1), config.prepared_statements_cache_size);
-        assertNull(config.key_cache_size);
+        if (config.key_cache_size != null)  // null in default test config, 0 in latest
```

Comment:
`latest what?` is what came to mind when I read that. Add ticket number to the reader can see `latest` refers to a special config set-up. wdyt?

Reply:
Done